clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -110,4 +109,10 @@ object ElasticsearchIndex extends ModelAutoDerivation {
       WgsCramIndex,
       ElasticsearchFieldMapper.StringsToTextFieldsWithSubKeywords
     )
+
+  implicit val Arrays: ElasticsearchIndex[ArraysIndex.type] =
```

Comment:
This (and the other indexes) are implicit, but as far as I can tell we aren't actually using it implicitly anywhere... we should either remove the `implicit` modifier or change `IndexService` to take the ES index as an implicit.